src/commandsAndMenu.tsx

```diff
@@ -87,6 +87,8 @@ export namespace CommandIDs {
   export const gitIgnoreExtension = 'git:context-ignoreExtension';
 }
 
+export const SUBMIT_COMMIT_COMMAND = 'git:submit-commit';
```

Comment:
Happy to add to the enum in #877, did not add to namespace to avoid annoying conflicts.

Reply:
Yes please do.